package excecao.personalizadaB;

import java.util.Scanner;

public class Finally {

	public static void main(String[] args) {

		Scanner tcd = new Scanner(System.in);
		try {
			System.out.println(7 / tcd.nextInt());
			tcd.close();

		} catch (Exception e) {
			System.out.println(e.getMessage());
		}finally {
			System.out.println("Finalmente...");
		}

		System.out.println("Fim!");
	}

}
